app.js

```diff
@@ -36,13 +37,25 @@ if(process.env.NODE_ENV === 'development'){
     app.use(morgan('dev'));
 }
 
+// ** Method Overrride middleware put and delete 
```

Comment:
use HTTP verbs such as PUT or DELETE in places where the client doesn't support it

Reply:
ok